### PR TITLE
fix: Typo in the notification invitation message

### LINKED ISSUE #5872: Typo in the notification invitation message
**Describe the bug**

Currently, one user gets the notification-<b>You've been invited to be `a` attendee at TEST EVENT2.</b> But for attendees and for organizer, it should be changed to-<b> You've been invited to be `an` attendee at TEST EVENT2.</b> according to standards.

**To Reproduce**
Steps to reproduce the behaviour:
1. Go to 'notification panel.'
2. Scroll down.
4. See error.

**Expected behaviour**

 Typo needs to be fixed for the case of attendees and organizer.

**Screenshots**

Screenshot 2019-04-20 17:21:55

**Desktop (please complete the following information):**
 - OS: Ubuntu 18.04
 - Browser Chrome
 - Version71.0398

**Additional context**

Working on it.

### PR DESCRIPTION
**Pull request by smcri**
Fixes #5872 

#### Short description of what this resolves:

Resolves display of 'a' in event notifications

#### Changes proposed in this pull request:

Changes notification message such that event role beginning with vowel or consonant does not matter

-
-
-

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
- [x] My branch is up-to-date with the Upstream `development` branch.
- [ ] The unit tests pass locally with my changes 
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

- [ ] All the functions created/modified in this PR contain relevant docstrings.

### CODE REVIEW

File: app/api/helpers/system_notifications.py [hunk 1]
```diff
@@ -392,7 +392,7 @@ def get_invite_papers_notification_actions(cfs_link, submit_link):
     },
     EVENT_ROLE: {
         'title': u'Invitation to be {role_name} at {event_name}',
-        'message': u"You've been invited to be a <strong>{role_name}</strong>" +
+        'message': u"You've been invited to be one of the <strong>{role_name}</strong>s" +
```
Comment (shreyanshdwivedi) [inline app/api/helpers/system_notifications.py]: Just replace `a` with `an` as mentioned in the issue

Comment (kushthedude) [reply app/api/helpers/system_notifications.py]: @shreyanshdwivedi It will require a condition block too then, Its a nice makeover done by him 😄

Comment (shreyanshdwivedi) [reply app/api/helpers/system_notifications.py]: Then the `s` must be in `strong` block itself

**Comment by smcri** [comment]
Moved `s` literal inside `strong` block.

**Comment by smcri** [comment]
Unable to merge into `fossasia: development`.

**Comment by mrsaicharan1** [comment]
You need to have 2 approvals from people with write access. Only then they
can merge it.

**Pull request merged**